examples/nonroot/BUILD

```
@@ -40,6 +40,7 @@ container_image(
 go_binary(
     name = "user",
     srcs = ["testdata/user.go"],
+    goos = "linux",
```

I think it's worth copying your comment here too.
```suggestion
    # Test image is linux based
    goos = "linux",
```